### PR TITLE
Build MinGW on travis

### PR DESCRIPTION
**Pull request by selaux**

### CODE REVIEW

**Comment by selaux** [comment]
This is finished from my side. I didn't test the resulting package yet though.

I also fixed that when the upload fails the build fails. Now it should succeed either way. Not sure what is wrong there, might be an IP blacklist or something, because it is happening reproducibly only for the linux builds.

File: .travis.yml [hunk 1]
```diff
@@ -58,20 +75,28 @@ script:
...
-  - if [ "$PUBLISH_BINARY" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      make rust-unit-tests;
+    fi
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      ./ja2 -unittests;
+    fi
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      ./ja2-launcher -help;
+    fi
```
Comment (lynxlynxlynx) [inline .travis.yml]: These three could be in the same block, but it's not that important. The diff looks fine, so merge away, unless you want to wait for testers/testing.

Comment (selaux) [reply .travis.yml]: Merged them...

Comment (lynxlynxlynx) [reply .travis.yml]: Are you sure they don't need to be and-chained (eg. make failing, launcher passing -> build passing)?

Comment (selaux) [reply .travis.yml]: True. Fixed it on master...

**Pull request merged**